### PR TITLE
Setup clientcore perf pipeline

### PR DESCRIPTION
**Pull request by samvaity**

### CODE REVIEW

**Review by g2vinay — APPROVED** [review]

File: sdk/clientcore/perf-tests.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Service: clientcore
```
Comment (weshaggard) [inline sdk/clientcore/perf-tests.yml:1]: Where is this file used?

Comment (samvaity) [reply sdk/clientcore/perf-tests.yml:1]: I think we use it here @weshaggard to run perf tests https://github.com/Azure/azure-sdk-for-java/blob/main/eng/common/pipelines/templates/jobs/perf.yml#L133

**Pull request merged**